test/Modular Design/Reports/HIV Dashboard Helpers/CD4_Totals.js

```diff
@@ -0,0 +1,70 @@
+const Encounters = require("../../Encounters");
+
+class CD4_Totals {
+    #DISAGGREGATION = {
```

Comment:
same for the casing, should be lower case

Reply:
Hi @nour-borgi , thanks for the review. i have attended to all comments. Please can you review again? if happy, I think @michaelloosen will have to be the one to approve and merge?